pipelines/docker-build-rhtap/patch.yaml

```diff
@@ -8,6 +8,37 @@
     "pipelines.openshift.io/used-by": "build-cloud"
     "pipelines.openshift.io/runtime": "generic"
     "pipelines.openshift.io/strategy": "docker"
+# Remove unused parameters from the template
```

Comment:
@chmeliik @mmorhun , I have removed unneeded parameters from the RHTAP pipeline in this PR. Let me know if you want to get these in with a different PR.